### PR TITLE
[l10n_id_accountant_report_stakeholder_report_bpk] 8.0.1.0.0

### PR DESCRIPTION
**Pull request by azmimr67**
* Task 8.0-T005297
* Pembuatan Awal
* Revisi sesuai github
* Pindah repo

### CODE REVIEW

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_sequence_configurator_data.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            />
+</record>
+
+<record id="base_sequence_stakeholder_bpk_line" model="base.sequence_configurator_line">
+    <field
+                name="generator_id"
+                ref="l10n_id_accountant_report_stakeholder_report_bpk.base_sequence_stakeholder_bpk"
+            />
+    <field name="sequence" eval="1" />
+    <field name="sequence_computation_code">result = False
```
Comment (mikevhe18) [inline l10n_id_accountant_report_stakeholder_report_bpk/data/base_sequence_configurator_data.xml]: "**result = False**" dihapus

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        >
+    <field
+                name="workflow_id"
+                ref="l10n_id_accountant_report_stakeholder_report_bpk.base_workflow_accountant_report_stakeholder_report_bpk"
+            />
+    <field
+                name="field_id"
+                search="[('model_id.model','=','accountant.report_stakeholder_report_bpk'),('name','=','cancel_ok')]"
+            />
+    <field name="python_code">result = allowed_wokflow_policy.cancel_grp_ids.ids
```
Comment (mikevhe18) [inline l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml]: result = document.type_id.cancel_grp_ids.ids

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        >
+    <field
+                name="workflow_id"
+                ref="l10n_id_accountant_report_stakeholder_report_bpk.base_workflow_accountant_report_stakeholder_report_bpk"
+            />
+    <field
+                name="field_id"
+                search="[('model_id.model','=','accountant.report_stakeholder_report_bpk'),('name','=','restart_ok')]"
+            />
+    <field name="python_code">result = allowed_wokflow_policy.restart_grp_ids.ids
```
Comment (mikevhe18) [inline l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml]: result = document.type_id.restart_grp_ids.ids

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+                name="workflow_id"
+                ref="l10n_id_accountant_report_stakeholder_report_bpk.base_workflow_accountant_report_stakeholder_report_bpk"
+            />
+    <field
+                name="field_id"
+                search="[('model_id.model','=','accountant.report_stakeholder_report_bpk'),('name','=','restart_validation_ok')]"
+            />
+    <field
+                name="python_code"
+            >result = allowed_wokflow_policy.restart_validation_grp_ids.ids
```
Comment (mikevhe18) [inline l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml]: result = document.type_id.restart_validation_grp_ids.ids

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        >
+    <field
+                name="workflow_id"
+                ref="l10n_id_accountant_report_stakeholder_report_bpk.base_workflow_accountant_report_stakeholder_report_bpk"
+            />
+    <field
+                name="field_id"
+                search="[('model_id.model','=','accountant.report_stakeholder_report_bpk'),('name','=','confirm_ok')]"
+            />
+    <field name="python_code">result = allowed_wokflow_policy.confirm_grp_ids.ids
```
Comment (mikevhe18) [inline l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml]: result = document.type_id.confirm_grp_ids.ids

File: l10n_id_accountant_report_stakeholder_report_bpk/README.rst [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+BPK Stakeholder Report for Accountant Report
+============================================
+
+
+Installation
+============
+
+To install this module, you need to:
+
+1.  Clone the branch 8.0 of the repository https://github.com/open-synergy/opnsynid-vertical-accountant
```
Comment (mikevhe18) [inline l10n_id_accountant_report_stakeholder_report_bpk/README.rst]: https://github.com/open-synergy/opnsynid-l10n-indonesia-vertical-accountant

File: l10n_id_accountant_report_stakeholder_report_bpk/README.rst [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+3.  Update the module list
+4.  Go to menu *Setting -> Modules -> Local Modules*
+5.  Search For *BPK Stakeholder Report for Accountant Report*
+6.  Install the module
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/open-synergy/opnsynid-vertical-accountant/issues>`_.
```
Comment (mikevhe18) [inline l10n_id_accountant_report_stakeholder_report_bpk/README.rst]: https://github.com/open-synergy/opnsynid-l10n-indonesia-vertical-accountant

**Review by mikevhe18 — CHANGES_REQUESTED** [review]

**Comment by mikevhe18** [comment]
**NOTE:** Tolong jangan lupa untuk dicoba dulu, terutama untuk modul policy, multiple approval, dll.

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_sequence_configurator_data.xml (hunk 1 above)
Comment (azmimr67) [reply l10n_id_accountant_report_stakeholder_report_bpk/data/base_sequence_configurator_data.xml]: done

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml (hunk 2 above)
Comment (azmimr67) [reply l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml]: done

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml (hunk 4 above)
Comment (azmimr67) [reply l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml]: done

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml (hunk 3 above)
Comment (azmimr67) [reply l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml]: done

File: l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml (hunk 5 above)
Comment (azmimr67) [reply l10n_id_accountant_report_stakeholder_report_bpk/data/base_workflow_policy_stakeholder_report_bpk_data.xml]: done

File: l10n_id_accountant_report_stakeholder_report_bpk/README.rst (hunk 6 above)
Comment (azmimr67) [reply l10n_id_accountant_report_stakeholder_report_bpk/README.rst]: done

File: l10n_id_accountant_report_stakeholder_report_bpk/README.rst (hunk 7 above)
Comment (azmimr67) [reply l10n_id_accountant_report_stakeholder_report_bpk/README.rst]: done

**Pull request closed without merging**